samples/client/petstore/k6/script.js

```diff
@@ -39,7 +39,7 @@ export default function() {
         sleep(SLEEP_DURATION);
     });
     group("/pet/findByStatus", () => {
-        let status = "TODO_EDIT_THE_STATUS";
+		let status = 'TODO_EDIT_THE_STATUS'; // specify value as there is no example value for this parameter in OpenAPI spec
```

Comment:
The extra indentation in the template is reflected here, which I think is unnecessary.